test/msg_cache_unittest.cpp

```diff
@@ -75,6 +75,18 @@ void fillCacheEasy(message_filters::Cache<Msg> & cache, unsigned int start, unsi
   }
 }
 
+TEST(Cache, emptySurroundingInterval)
+{
+  message_filters::Cache<Msg> cache(10);
+  const std::vector<std::shared_ptr<Msg const>> interval_data = cache.getSurroundingInterval(
+    rclcpp::Time(
+      5,
+      0), rclcpp::Time(
+      9, 0));
```

Comment:
Indentation looks odd. Did you try something like this ?
```suggestion
    rclcpp::Time(5, 0), rclcpp::Time(9, 0));
```